e2e/__tests__/onlyChanged.test.ts

```diff
@@ -17,6 +18,14 @@ const HG = 'hg --config ui.username=jest_test';
 beforeEach(() => cleanup(DIR));
 afterEach(() => cleanup(DIR));
 
+// Certain environments (like CITGM and GH Actions) do not come with mercurial installed
+const hgIsInstalled = which.sync('hg', {nothrow: true}) !== null;
```

Comment:
should we stick this helper in `@jest/test-utils`?

Reply:
I can move it into `@jest/test-utils` if you want to. If it is the case, I will also move the code checking hg used by `jestChangedFiles.test.ts`.

There is also `import {cleanup, run, writeFiles} from '../Utils';` where we can put the helper? 🤔

Reply:
Yeah, sticking in in Utils is probably cleaner, let's do that! 🙂